sparkle/src/components/Avatar.tsx

```diff
@@ -199,8 +199,9 @@ Avatar.Stack = function ({
         marginLeft: "20px",
       }}
     >
-      {childrenArray.map((child) => (
+      {childrenArray.map((child, i) => (
         <div
+          key={i}
```

Comment:
Since there is an image in there ideally we don't rerender on agent addition so maybe we can use something more precise? if not that's fine

Reply:
If we push a new participant it will be at the end of the array, introducing a new item but not moving indexes of previous participants, so not rendering again. So unless I'm missing something we should be good!